homeassistant/components/light/__init__.py

```diff
@@ -99,7 +104,7 @@
                                             max=color_util.HASS_COLOR_MAX)),
     ATTR_WHITE_VALUE: vol.All(int, vol.Range(min=0, max=255)),
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
-    ATTR_EFFECT: vol.In([EFFECT_COLORLOOP, EFFECT_RANDOM, EFFECT_WHITE]),
+    ATTR_EFFECT: str,
```

Comment:
Some other ATTR_ in the same block are using `str`. Why `cv.string` here?
See [`ATTR_COLORNAME`](https://github.com/home-assistant/home-assistant/pull/4538/files/4c4eb757f12f01abd5e3f98c5f904419bee0adcf#diff-685ba8a5bcd2c24b5af57da8529e8ce9L93) for example.